frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -208,27 +212,20 @@ export class Wizard {
     if (description) {
       await this.fillDescription(description);
     }
-    if ((await browser.getCurrentUrl()).match(/\?template=.+$/)) {
-      // We are creating a VM from template via its action button
-      // ProvisionSource, OS and workload are prefilled and disabled - ignoring them
-    } else if (template) {
-      await this.selectTemplate(template);
+    if (os) {
```

Comment:
please can you show me, where is the logic for the template now? We are still offering this option.

Reply:
The wizard field is removed by [1871103](https://bugzilla.redhat.com/show_bug.cgi?id=1871103), so remove it here as well.

Reply:
the field was removed, but the wizard is still used for creating VMs from the template. So it should still not be possible to select os/workload/etc for these VMs. This applies for other logic as well. Only the selection should be removed.

Still we could automate this and add the template field tot the build and it would just go to the template page and select the create VM action

Reply:
Removed field template works well for create vm from template or not from template based on the new changes and the tests are PASS for vm.wizard and vmtemplate.wizard:
- template is removed in vm wizard
- the new link/button to create vm from template